README.md

```diff
@@ -2,16 +2,41 @@
 
 [![Streamlit App](https://static.streamlit.io/badges/streamlit_badge_black_white.svg)](https://blackary-mpa-config-streamlit-app-0bp2ol.streamlitapp.com/)
 
-An example of how to use a page_config.yaml file to control the ordering, icons, and nesting
-of the pages in the sidebar of a Multi-Page Streamlit App.
+<img width="600" src="https://user-images.githubusercontent.com/7164864/178483961-1f9abf49-eb44-455f-9362-37951daf055a.gif">
 
-Features include:
+### Summary 
 
-- Controling the order of the pages displayed
-- Nesting related apps within a "section"
-- Adding a created date on each app, and automatically adding a 🆕 icon
-- Automatically adding the icon and title at the top of each page
-- Displaying some pages in wide-mode
+Streamlit recently [released multi-page apps](https://blog.streamlit.io/introducing-multipage-apps/) 🎉 where page filenames are the source of truth for page settings. 
 
-Note that this depends on an API that may change, and is not designed to be used this way,
+In this repository, we show a prototype on how to use a [page_config.yaml](https://github.com/blackary/mpa-config/blob/main/page_config.yaml) to control the ordering, icons, and nesting of the pages in the sidebar of a multi-page Streamlit app.
+
+Main features include:
+
+- **Decoupling page settings from filenames.** No need to use indices or emojis in filenames to handle page titles, icons or ordering! You can name your Python scripts however you want.
+- **Controling the order of the pages.** Adding pages in the config in the order you want them to appear in the app.
+- **Nesting related pages within a "section".** Using `sections` in page_config.yaml will add un-clickable placeholders that group pages together.
+- **Automatically adding the icon and title at the top of each page.** Bringing consistency to your pages!
+
+⚠️ This project depends on an API that may change, and is not designed to be used this way,
 so this will probably NOT work long-term.
+
+
+### Get started
+
+1. Install requirements `pip install -r requirements.txt` 
+2. Run the Streamlit app `streamlit run streamlit_app.py`
+3. Play with the page_config.yaml!
+
+
+### Documentation
+
+Supported parameters in page_config.yaml for `pages`:
+- page_name: Page name.
```

Comment:
What about calling that `name` btw?

Reply:
I don't have a strong preference

Reply:
Two cents here:
- `name` would be more consistent with how `sections` are called. And also `pages_` prefix is not used elsewhere
- `page_name` is used in `st.set_page_config()` so I also understand the temptation. But with it then comes `page_icon`... aaaaah letting you choose

Reply:
Oh, but @blackary isn't `page_config` the expected naming from Streamlit internals? Maybe better stick to it then. Tried naive changes but it's breaking the thing

Reply:
Oh, but @blackary isn't `page_config` the expected naming from Streamlit internals? Maybe better stick to it then. Tried naive changes but it's breaking the thing

Reply:
Yeah, it probably makes sense to keep it as `page_name`, if only because that's ultimately the name it gets in the canonical page list. Doesn't matter too much, because we can change it easily enough, but if you don't feel too strongly, let's leave it as page_name.